### PR TITLE
Improve Github Actions

### PR DESCRIPTION
**Pull request by jordisala1991**
Now they look similar to other sonata-project bundles

### CODE REVIEW

**Comment by OskarStark** [comment]
Do we really want to have PHP 8 test?

**Comment by jordisala1991** [comment]
IMO it is nice to have it, one day we would like to switch, and having test with it helps the migration (even if it does not work now, probably because some incompatibility with SF)

File: config/packages/github_api.yaml [hunk 1]
```diff
@@ -3,7 +3,9 @@ services:
         arguments:
             - '@Github\HttpClient\Builder'
         calls:
-            - ['authenticate', ['%env(GITHUB_OAUTH_TOKEN)%', null, !php/const \Github\Client::AUTH_HTTP_TOKEN]]
+            # use: !php/const \Github\Client::AUTH_HTTP_TOKEN instead of: 'http_token'
+            # when the yaml parser can deal with custom tags
+            - ['authenticate', ['%env(GITHUB_OAUTH_TOKEN)%', null, 'http_token']]
```
Comment (jordisala1991) [inline config/packages/github_api.yaml:8]: Had to do this change because of the yaml-parser, it is bad that it does not support custom tags

**Comment by OskarStark** [comment]
`Test / PHP 7.4 (pull_request)` should be required

**Comment by jordisala1991** [comment]
This has to be configured on the settings of this repository, I dont have enough rights to do it

**Pull request merged**